### DISCUSSION TITLE
How to train on ADNI dataset

### DISCUSSION THREAD

**Discussion by fadyben98** [aramis-lab/clinicadl#303] [Q&A] [answered]
Hi,

My goal is to experiment with deep learning models to perform multi-class classification (CN, AD, pMCI, sMCI) on ADNI data. Your library seems perfect for this but I can't figure out how to do this. 
- Which ADNI dataset should I download (ADNI_baseline, ADNI 1 Complete 1Yr, ADNI Complete 2Yr, ADNI Complete 3Yr, something else)?
- How can I get the labels (CN, AD, pMCI, sMCI) along with the input data (Tensor for each MRI image)?

I tried reading the docs but it's unclear for me how I can do this. I would greatly appreciate more details on how I can do this. Thanks!

**Comment by ravih18** [comment, marked as answer]
Hello @fadyben98 ,

Thanks for your interest in ClinicaDL. Indeed ClinicaDL will provide you some great tools for multi-class classification on ADNI.

Concerning which ADNI dataset you should use, you should maybe look at [Clinica software](https://aramislab.paris.inria.fr/clinica/docs/public/latest/). Especially the [documentation of ADNI-2-BIDS converter](https://aramislab.paris.inria.fr/clinica/docs/public/latest/Converters/ADNI2BIDS/).

Moreover, Clinica's pipeline will provide you a TSV file with all the labels for each session (i.e. for each image).

Once you have converted the ADNI dataset you need to BIDS format, you can run [Clinica T1-linear pipeline](https://aramislab.paris.inria.fr/clinica/docs/public/latest/Pipelines/T1_Linear/) to preprocess your data. And then only you can use ClinicaDL to perform your deep-learning study.

Regards, 

Ravi
